.github/workflows/Build-Rock.yaml

```diff
@@ -72,6 +72,20 @@ jobs:
             import yaml
             import os
 
+            def string_constructor(loader, node):
+              return loader.construct_scalar(node)
+            
+            yaml.add_constructor(
+                "tag:yaml.org,2002:int",
+                string_constructor,
+                Loader=yaml.SafeLoader
+            )
+            yaml.add_constructor(
+                "tag:yaml.org,2002:float",
+                string_constructor,
+                Loader=yaml.SafeLoader
+            )
+
```

Comment:
why do we need this here? afaiu this code is only using the `platforms` field, which can never be a number.

Reply:
Safe precaution for the future. If you deem it unncessary I'll remove it.

Reply:
let's remove it cause it's not doing anything but bloating an already fragile piece of code